pages/about.html

```diff
@@ -1,12 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Kimia Hasani, Front-end & Back-end Developer</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Caveat&display=swap" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" rel="stylesheet">
+  <link rel="stylesheet" href="/style/style.css">
```

Comment:
Careful with your file paths! The `/style/style.css` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `../styles/style.css`. The two dots at the beginning signal to start searching in the parent directory of the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This applies to links to other pages on your website and image URLs as well.